src/components/UI/Cards/LikedVideoCards/LikedVideoCards.jsx

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useLikes } from '../../../../context/LikesContext';
+import { VAR_ENCODE_TOKEN } from '../../../../utils/Route';
```

Comment:
Great you are storing the route path in a variable. This is new to me. Great work.